packages/engine-core/src/tools/byline.ts

```diff
@@ -1,3 +1,4 @@
+/* eslint-disable @typescript-eslint/no-this-alias */
```

Comment:
If the content differs, it would be nice to diff `byline.ts` files across the packages to see if they are functionally equivalent.

Hopefully the difference was only in the comments and type annotations, but if there are different versions of the code, we don't really know which one was used, because these modules weren't isolated and would override each other's `createStream` and `LineStream` on the bundle's `module.exports`.

Reply:
They should be the same besides some comments I think